### PR TITLE
Feature/photo from outlook

### PR DESCRIPTION
**Pull request by MMSaavedraC**

### CODE REVIEW

File: consultantmarket/settings.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 SECRET_KEY = 'o&nx!s$tilel2kn7^$1%4vdo*g5zpu9+e4d8*668w(oj!99jde'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (mmartinezintersys) [inline consultantmarket/settings.py]: Remember to keep the False state when pushing the code

File: consultantmarket/settings.py [hunk 2]
```diff
@@ -182,8 +182,8 @@
     pass
 
 EMAIL_HOST = 'smtp-mail.outlook.com'
-EMAIL_HOST_USER = 'intersysinternalapplication@intersysconsulting.com'
-EMAIL_HOST_PASSWORD = 'Internal2018!'
+EMAIL_HOST_USER = 'intersysinternalapp@intersysconsulting.com'
+EMAIL_HOST_PASSWORD = '1nt3rn5ys4pp#19'
```
Comment (mmartinezintersys) [inline consultantmarket/settings.py]: This has changed in develop so do not forget to keep the new way to get this values from the properties file

File: consultantmarket/urls.py [hunk 3]
```diff
@@ -25,5 +25,9 @@
     path('admin/', admin.site.urls),
     path('noaccess/', noAccess, name="noAccess"),
     path('credits/', credits, name="credits"),
+    path('faq/', faq, name="faq"),
```
Comment (mmartinezintersys) [inline consultantmarket/urls.py]: The reference to the faq page is not in develop do not be push it to develop

**Comment by MMSaavedraC** [comment]
Made a mistake when resolving conflicts, created another branch so I could merge. Will close this pr and delete the branch.

**Pull request closed without merging**